blog/2015/11/swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,436 @@
+<a href='http://pad.software-carpentry.org/swc-instructor-retreat-2015-7PM-UTC'>read</a> the notes on etherpad.
+
+<!-- end excerpt -->
+
+First, thanks to everyone who joined the discussion:
+<ul>
+  <li>Tiffany Timbers / @TiffanyTimbers</li>
+  <li>Jenny Bryan / @JennyBryan </li>
+  <li>Daniel Chen / @chendaniely </li>
+  <li>Camille Avestruz</li>
+  <li>David LeBauer</li>
+  <li>Cam Macdonell</li>
+  <li>Drew Tyre / @atiretoo</li>
+  <li>Rayna Harris / @raynamharris and the +5 from <a href='https://twitter.com/choleness/status/665613152181092352'>Austin Texas</a> including: Rebecca Tarvin / @frogsicles</li>
+  <li>Jon Pipitone</li>
+  <li>John Moreau</li>
+  <li>Gayathri Swaminathan</li>
+  <li>Nichole Bennett</li>
+  <li>Tracy Teal</li>
+  <li>Raniere Silva</li>
+  <li>Sarah Stevens</li>
+  <li>Lex Nederbragt</li>
+  <li>Kate Hertweck</li>
+  <li>Greg Wilson</li>
+  <li>Becca Tarvin</li>
+  <li>Azalee Bostroem</li>
+</ul>
+
+Hopefully I got everyone!
+
+<h2>Motivation</h2>
+Last year I <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002202.html'>asked</a> the <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/'>discuss list</a>
+if anyone had any experience with turing the <a href='http://software-carpentry.org/lessons.html'>software-carpentry lessons</a> into a university course.
+Bill Mills <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002203.html'>mentioned</a> that two main hurdles revolve around politics and curriculum control.
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002204.html'>Ethan White</a>
+and <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002211.html'>Michael Jackson</a>went on to describe their own experiences with settin up a course.
+I want to believe the push for code sharing and reproducibility article from
+<a href='http://www.nature.com/news/interactive-notebooks-sharing-the-code-1.16261'>Nature</a> and submission guidelines for
+<a href='http://journals.plos.org/plosone/s/materials-and-software-sharing'>PLOS ONE</a>
+have addressed <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002205.html'>Greg's</a> initial observations that universites find that lab skills course is worth graduate credit.  The conversation contines
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002207.html'>here</a>,
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002210.html'>here</a>, and
+a blog post by Damien Irving <a href='http://software-carpentry.org/blog/2014/02/university-course.html'>here</a>.
```

Comment:
“The conversation continues here…” is not a very strong inducement to go read the referenced articles.  What points are being raised in those messages that make them worth citing?  Can you condense that down to a few words or a phrase that could live in the link text?